daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -0,0 +1,85 @@
+---
+type: docs
+title: Multi-app Run overview
+linkTitle: Multi-App Run overview
+weight: 1000
+description: Learn the scenarios around running multiple applications with one CLI command
+---
+
+{{% alert title="Note" color="primary" %}}
+ Multi-App Run is currently a preview feature only supported in Linux/MacOS. 
+{{% /alert %}}
+
+Let's say you want to run several applications locally to test them together, similar to a production scenario. With a local Kubernetes cluster, you'd be able to do this with helm/deployment YAML files. You'd also have to build them as containers and set up Kubernetes, which can add some complexity. 
+
+Instead, you simply want to run them as local executables in self-hosted mode.  However, self-hosted mode requires you to:
+
+- Run multiple `dapr run` commands
+- Keep track of all ports opened (you cannot have duplicate ports for different applications).
+- Remember the resources folders and configuration files that each application refers to.
+- Recall all of the additional flags you used to tweak the `dapr run` command behavior (`--app-health-check-path`, `--dapr-grpc-port`, `--unix-domain-socket`, etc.)
+
+With Multi-app Run, you can start multiple applications in self-hosted mode using a single `dapr run -f` command using a template file. The template file describes how to start multiple applications as if you had run many separate CLI `run`commands. By default, this template file is called `dapr.yaml`. 
```

Comment:
should it be `Multi-App` or `Multi-app`? I think we should be consistent across the doc.

Reply:
thanks for catching - I'll go with Multi-App